include/openenclave/bits/seal.h

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Open Enclave SDK contributors.
```

Comment:
Ping. This comment remains unaddressed?

Reply:
@CodeMonkeyLeet , `bits/seal.h` is included by `enclave.h`, so developers are not expected to include it directly. I'd like to keep the current user experience of "including `enclave.h` includes everything" unchanged, so it may look inconsistent to move it one level up (headers included by `enclave.h` are all in `bits/` so far). Another option is to merge `bits/seal.h` into `enclave.h` but given all the macro definitions, the merged `enclave.h` will look awkward. So I'm going to keep it as a separate `seal.h` under `bits/`, unless you insist.